cmd/config/virt-migration/udn_l2.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: l2-network-migration
+spec:
+  topology: Layer2
+  layer2:
+    role: Primary
+    subnets: ["10.132.0.0/24"]
```

Comment:
Could using a hardcoded value here cause an issue if the cluster or environment use this CIDR somewhere else?